sqldb/migrations_test.go

```diff
@@ -570,3 +570,164 @@ func TestSchemaMigrationIdempotency(t *testing.T) {
 		}
 	})
 }
+
+// TestMigrationBug19RC1 tests a bug that was present in the migration code
+// at the v0.19.0-rc1 release.
+// The bug was fixed in: https://github.com/lightningnetwork/lnd/pull/9647
+// NOTE: this test may be removed once the funal version of 0.19.0 is released.
```

Comment:
Micro Nit: NOTE: This ...

funal => final ?

Reply:
Thanks :)